src/session_stats.cpp

```diff
@@ -199,6 +199,13 @@ namespace libtorrent
 		// NAT or not.
 		METRIC(net, has_incoming_connections)
 
+		METRIC(net, total_download_rate)
```

Comment:
could add documentation here as well (these comments are actually used to generate the documentation)